blog/2022-09-12-psp-deprecation.mdx

```diff
@@ -0,0 +1,50 @@
+---
+title: KubeSlice Security and PSP Removal
+description: Blog post about KubeSlice Security and PSP Removal
+slug: kubeslice-security-psp-removal
+authors:
+  - name: Leon Nunes
+    url: https://github.com/afro-coder
+hide_table_of_contents: false
+---
+
+In Kubeslice security is something that is Step 0 rather than an afterthought.
+
+#### What are Pod Security Policies?
+> PodSecurityPolicy is a built-in [admission controller](https://kubernetes.io/blog/2019/03/21/a-guide-to-kubernetes-admission-controllers/) that allows a cluster administrator to control security-sensitive aspects of the Pod specification.
+One or more PodSecurityPolicy resources are created in a cluster to define the requirements Pods must meet. Then, RBAC rules are created to control which PodSecurityPolicy applies to a given pod. If a pod meets the requirements of its PSP, it will be admitted to the cluster as usual.
+
+**Note**: The [Kubernetes 1.25 release](https://kubernetes.io/blog/2022/08/04/upcoming-changes-in-kubernetes-1-25/) will remove PSP so if you rely on PSP you should follow the instructions for [migration to Pod Security Admission](https://kubernetes.io/docs/tasks/configure-pod-container/migrate-from-psp/)
+
+### How KubeSlice implements Security and PSS?
+KubeSlice segregates the Network in Slices as tenants shouldn't need to communicate with all the resources in the cluster.
```

Comment:
```suggestion
In KubeSlice security is something that is Step 0 rather than an afterthought. 
KubeSlice segregates the Network in Slices as tenants shouldn't need to 
communicate with all the resources in the cluster.
```